newIDE/app/src/MainFrame/ElectronMainMenu.js

```diff
@@ -85,6 +86,13 @@ const buildAndSendMenuTemplate = (project, i18n) => {
         accelerator: 'CommandOrControl+O',
         onClickSendEvent: 'main-menu-open',
       },
+      {
+        label: i18n._(t`Open Recent`),
+        submenu: recentProjectFiles.map(item => ({
+          label: item.fileIdentifier,
+          onClickSendEvent: 'main-menu-open-recent',
```

Comment:
This wouldn't work obviously, because the function expects `fileMetadata` as argument.

Reply:
We probably need to extend the `onClickSendEvent` so that it also send an optional value stored in a new `eventArgument` property: https://github.com/4ian/GDevelop/blob/93e0ccc1634972b42a47f7c2e5c770b310900c82/newIDE/electron-app/app/MainMenu.js#L24-L26

Then, I think `useIPCEventListener` is already calling the function with the arguments passed to the event from Electron.